admin/acceptance.py

```diff
@@ -277,6 +323,17 @@ def stop_nodes(self, reactor):
 PROVIDERS = tuple(sorted(['vagrant'] + CLOUD_PROVIDERS.keys()))
 
 
+class TestTypes(Values):
+    """
+    Type of test to run.
+
+    :ivar CLIENT: Run client smoke tests.
```

Comment:
I think that calling them "installation tests" for now, until they are potentially changed, would be clearer.